netlify.toml

```diff
@@ -1,8 +1,8 @@
 [dev]
-  command = "npm run dev"
+  command = "bun run dev"
```

Comment:
Thoughts on using pnpm or yarn instead? It is faster to my knowledge and has more functionallity?

Reply:
it is? i'm not really an expert on different package managers; i mostly just use bun

Reply:
its faster then npm but not bun

Reply:
let's do bun